plcc/src/io/CifParser.java

```diff
@@ -798,7 +881,7 @@ private static void handleAtomSiteLine() {
                 rna.setType(Molecule.RESIDUE_TYPE_RNA);                
                 
                 RnaTreatedNum++;
-                int resNumDSSP = DsspParser.lastUsedDsspNum + RnaTreatedNum + ligandsTreatedNum;
+                int resNumDSSP = DsspParser.lastUsedDsspNum + RnaTreatedNum + ligandsTreatedNum + freeResTreatedNum;
```

Comment:
How many times do we have such a line where we need a new fake DSSP num? If more than once, I suggest we create a new function so that if in future someone adjusts it (another category like free residues or new mol types, such as carbohydrates) it stays consistent.